ibis/backends/bigquery/compiler.py

```diff
@@ -130,16 +130,18 @@ def _quote_identifier(self, name):
         return f"`{name}`"
 
     def _format_in_memory_table(self, op):
+        import ibis
+
         schema = op.schema
         names = schema.names
         types = schema.types
 
         raw_rows = []
         for row in op.data.to_frame().itertuples(index=False):
             raw_row = ", ".join(
-                f"{self._translate(lit)} AS {name}"
+                f"{self._translate(lit.op())} AS {name}"
                 for lit, name in zip(
-                    map(ops.Literal, row, types), map(self._quote_identifier, names)
+                    map(ibis.literal, row, types), map(self._quote_identifier, names)
```

Comment:
Why is this change necessary?

Reply:
Because `ops.Literal` doesn't normalize `list` inputs for array types. It's only accepts hashable sequences:

```
In [5]: import ibis

In [6]: ibis.literal(["a"])
Out[6]: ('a',)

In [7]: import ibis.expr.operations as ops

In [8]: ops.Literal(["a"], dtype="array<string>")
...
IbisTypeError: argument passes none of the following rules: ...
```

Reply:
Could you please try that out on the 7.x branch?